scvi/inference/inference.py

```diff
@@ -41,28 +41,48 @@ def __init__(
         train_size=0.8,
         test_size=None,
         n_epochs_kl_warmup=400,
+        normalize_loss=None,
         **kwargs
     ):
         super().__init__(model, gene_dataset, **kwargs)
         self.n_epochs_kl_warmup = n_epochs_kl_warmup
+
+        self.normalize_loss = (
+            not (
+                hasattr(self.model, "reconstruction_loss")
+                and self.model.reconstruction_loss == "autozinb"
+            )
+            if normalize_loss is None
+            else normalize_loss
+        )
+
+        self.n_scale = 1.0
```

Comment:
Can you comment what this is what why this has to take different values (testing set and training set) ? Maybe refer to the equation in your paper or the AEVB paper ?